src/app/components/Footer.tsx

```diff
@@ -37,6 +37,12 @@ const Footer = () => {
           </GridItem>
         </Grid>
       </Bullseye>
+      <Bullseye>
+        <Text component='small'>Disclaimer: This is an experimental project run by the Red Hat</Text>
```

Comment:
It's one `the` too much, right? :smile:

Reply:
Well its a two line sentence now the rest is below :D 
I still don't know how to make the line break work so I literally places them on top of each other. We need a front-end dev.

Reply:
.. but you know what? I think that one liner is more than enough.